iceberg/iceberg-handler/src/test/results/positive/col_stats.q.out

```diff
@@ -339,17 +339,16 @@ POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: default@tbl_ice_puffin
 col_name            	a                   
 data_type           	int                 
-min                 	1                   
-max                 	333                 
-num_nulls           	0                   
-distinct_count      	7                   
+min                 	                    
+max                 	                    
+num_nulls           	                    
+distinct_count      	                    
```

Comment:
This removed the iceberg column stats? If so we should revert and make sure it doesn't affect iceberg

Reply:
No, the `desc formatted tbl_ice_puffin a` doesn't fetch the stats from puffin files though with `hive.iceberg.stats.source=iceberg`, instead it goes to metastore for the stats.

the `tbl_ice_puffin` is an external table and recreated(inserted) multiple times before the `desc` query, this time when the table created, the legacy data files left behind make HMS believe that the column stats is stale, 
so stats of the insertion("values (1, 'one', 50), (2, 'two', 51),(2, 'two', 51),(2, 'two', 51), (3, 'three', 52), (4, 'four', 53)") after cann't be merged.

there is a `explain select * from tbl_ice_puffin order by a, b, c;` before the `desc`, as we can see, the stats stored in puffin files are not removed.

Reply:
This part of the output corresponds to the following code snippet.

```
set hive.iceberg.stats.source=iceberg;
drop table if exists tbl_ice_puffin;
create external table tbl_ice_puffin(a int, b string, c int) stored by iceberg tblproperties ('format-version'='2');
insert into tbl_ice_puffin values (1, 'one', 50), (2, 'two', 51),(2, 'two', 51),(2, 'two', 51), (3, 'three', 52), (4, 'four', 53), (5, 'five', 54), (111, 'one', 55), (333, 'two', 56);
explain select * from tbl_ice_puffin order by a, b, c;
select * from tbl_ice_puffin order by a, b, c;
select count(*) from tbl_ice_puffin ;
desc formatted tbl_ice_puffin a;
```

In this case the output of `desc formatted tbl_ice_puffin a; ` is accurate.

Reply:
This perhaps true for Iceberg table, as it will track its own metadata.  
Think about a native external table, if there are some files under the table directory on creation, then assumption probably is wrong(e,g min = 1, max=333) after insertion.

If every time we create a new Iceberg table in HMS, the legacy files under the table directory won't be read, e.g, the row number is 0 regardless of the legacy files, then we can put \"COLUMN_STATS\":{\"a\":\"true\",\"b\":\"true\",\"c\":\"true\" into table parameters.